Library/Homebrew/brew.sh

```diff
@@ -191,6 +191,16 @@ case "$HOMEBREW_COMMAND" in
   --config)    HOMEBREW_COMMAND="config" ;;
 esac
 
+if [[ -z "$HOMEBREW_DEVELOPER" ]]
+then
+  export HOMEBREW_GIT_CONFIG_FILE="$HOMEBREW_REPOSITORY/.git/config"
+  HOMEBREW_GIT_CONFIG_DEVELOPERMODE="$(git config --file="$HOMEBREW_GIT_CONFIG_FILE" --get homebrew.developermode)"
```

Comment:
Isn't this going to add overhead to every brew incantation?  I'm all for making the dev-cmd's available to anyone, but I'm not sure I see why there needs to be this git config file trick.  I can't think of why someone would want HOMEBREW_DEVELOPER *un*set but have something in the repo's gitconfig saying "I'm a developer".